catalog-rest-service/src/main/resources/json/schema/tests/basic.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
File name is tests/basic.json but id says test/basic.json

Reply:
@harshach, move this to a directory test instead of tests?

Reply:
going to remove basic.json all together. Currently don't need it